src/components/fields/location-field/location-modal/location-picker/location-picker.tsx

```diff
@@ -131,23 +143,61 @@ export const LocationPicker = ({
 		}
 	};
 
-	const zoomWithMarkers = (target: ILocationCoord) => {
+	const zoomWithMarkers = (
+		targets: ILocationCoord[],
+		shouldSetMarkers = true,
+		_zoomCenter?: ILocationCoord,
+		selectableMarkers = false,
+		enlargeSelectedMarker = false
+	) => {
```

Comment:
would there be a case where shouldSetMarkers is false, and selectableMarkers and enlargeSelectedMarker are also false?

Means dont show current location marker
yet there are no selectable pins.

what can the user do in this state?

Reply:
`enlargeSelectedMarker` shouldn't make a difference

`shouldSetMarkers` states whether there will be pins on the map

`selectableMarkers` state whether user can click on the pin

so if `shouldSetMarkers` is false, `selectableMarkers` won't make a difference since there is nothing to click anyway.

a separate setting in location-picker is whether or not user can click on map to select, so there is that option to select location